frontend/app/routes/_gcweb-app.new-application._index.tsx

```diff
@@ -1,50 +1,57 @@
 
 export const handle = {
-  breadcrumbs: [{ labelI18nKey: 'new-application:index.page-title' }],
+  breadcrumbs: [{ labelI18nKey: 'new-application:privacy-statement.breadcrumbs.new-application' }, { labelI18nKey: 'new-application:privacy-statement.breadcrumbs.privacy-statement' }],
   i18nNamespaces,
   pageIdentifier: 'CDCP-0015',
-  pageTitleI18nKey: 'new-application:index.page-title',
+  pageTitleI18nKey: 'new-application:privacy-statement.page-title',
 } as const satisfies RouteHandleData;
 
-export async function loader({ request }: LoaderFunctionArgs) {
-  const raoidcService = await getRaoidcService();
-  await raoidcService.handleSessionValidation(request);
-
-  const userService = getUserService();
-  const userId = await userService.getUserId();
-  const userInfo = await userService.getUserInfo(userId);
-  const countryList = await getLookupService().getAllCountries();
-  const regionList = await getLookupService().getAllRegions();
-
-  if (!userInfo) {
-    throw new Response(null, { status: 404 });
-  }
-
-  const preferredLanguage = userInfo.preferredLanguage && (await getLookupService().getPreferredLanguage(userInfo?.preferredLanguage));
-  const homeAddressInfo = userInfo.homeAddress && (await getAddressService().getAddressInfo(userId, userInfo?.homeAddress));
-  const mailingAddressInfo = userInfo.mailingAddress && (await getAddressService().getAddressInfo(userId, userInfo?.mailingAddress));
-
-  return json({ user: userInfo, preferredLanguage, homeAddressInfo, mailingAddressInfo, countryList, regionList });
-}
-
 export default function NewApplicationIndex() {
   const { t } = useTranslation(i18nNamespaces);
   return (
     <>
-      <p className="mb-8 text-lg text-gray-500">{t('new-application:index.subtitle')}</p>
+      <p className="mb-8 text-lg font-bold">{t('new-application:privacy-statement.privacy-statement.subtitle')}</p>
+      <p>{t('new-application:privacy-statement.privacy-statement.paragraph-one')}</p>
+      <br />
+      <p>{t('new-application:privacy-statement.privacy-statement.paragraph-two')}</p>
+      <br />
+      <p>{t('new-application:privacy-statement.privacy-statement.paragraph-three')}</p>
+      <br />
+      <p>{t('new-application:privacy-statement.privacy-statement.paragraph-four')}</p>
+      <br />
+      <br />
+
+      <p className="mb-8 text-lg font-bold">{t('new-application:privacy-statement.identity-information.subtitle')}</p>
+      <p>{t('new-application:privacy-statement.identity-information.paragraph-one')}</p>
+      <ul className="list-disc pl-10">
+        <li>{t('new-application:privacy-statement.identity-information.listitem-one')}</li>
+        <li>{t('new-application:privacy-statement.identity-information.listitem-two')}</li>
+      </ul>
+      <br />
+      <p>{t('new-application:privacy-statement.identity-information.paragraph-two')}</p>
+      <br />
+      <p>{t('new-application:privacy-statement.identity-information.paragraph-three')}</p>
+      <br />
+      <br />
```

Comment:
Avoid using `<br/>` for spacing. Use CSS instead (on the `<p>...</p>` - maybe something like `mb-8` class).